routes/web.php

```diff
@@ -13,6 +13,5 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-});
+Route::resource('/post', 'PostController');
+Route::resource('/tag', 'TagController');
```

Comment:
resource bỏ '/', đổi ra danh từ số nhiều em 
```suggestion
Route::resource('posts', 'PostController');
Route::resource('tags', 'TagController');
```

Reply:
ok anh